test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -126,4 +126,18 @@ Describe "Get-Content" -Tags "CI" {
     } else {$expected = "Hell","o,ll","ll","Worll","d`nHell","o2,ll","ll","Worll","d2`n"}
     for ($i = 0; $i -lt $result.Length ; $i++) { $result[$i]  | Should BeExactly $expected[$i]}
   }
+
```

Comment:
if you approach this by removing the dynamic parameter on ! windows, we should probably have an explicit test for the parameter - windows => parameter present ; ! windows => parameter absent